psamm/database.py

```diff
@@ -164,28 +164,67 @@ class DictDatabase(MetabolicDatabase):
 
```

Comment:
For this class, its parent class has some method like get_reaction(), may also need to change depending on what we want to keep in the database.